engine/src/main/java/org/hibernate/validator/constraints/SafeHtml.java

```diff
@@ -81,6 +81,12 @@
 		 * @return list of tag attributes which are whitelisted.
 		 */
 		String[] attributes() default { };
+
+		/**
+		 * @return list of valid protocols.
+		 * @since 6.0
+		 */
+		String[] protocols() default { };
```

Comment:
Hi @marko-bekhta ,

Sorry for being so late to the party. The BV release drained most of our workforce lately. I think there's a logic error here. Gunnar spotted it in his comment here (https://github.com/hibernate/hibernate-validator/pull/714/commits/9b14baa4d03b88789e7be478dbf6e0c80192df85#r110625190) but without underlining it: the protocols should be attached to the attributes and not to the tag.

The issue I can see with this is that we would break the compatibility if we make attribute its own annotation. I suppose we could follow the same logic as for the tags and add an `additionalAttributesWithProtocols` thingy. WDYT?